src/Utils/TemplateEngine.php

```diff
@@ -116,9 +116,13 @@ public function compile( $target, array $args = [] ) {
 
 		$complied = $this->container[$target];
 
-		foreach ( $args as $key => $value ) {
-			$complied = str_replace( [ '{{' . $key . '}}', '{{#' . $key . '}}' ], $value, $complied );
-		}
+		
+		foreach ($args as $key => $value) {
```

Comment:
Can you keep the existing code style and use tabs for indendation?

Reply:
Marked as unresolved since it's not resolved

Reply:
Not resolved. 

Note the spaces

```php
foreach ( $args as $key => $value ) {
```